app/components/fastboot-only/local-navigation-for-seo.js

```diff
@@ -0,0 +1,35 @@
+import Ember from 'ember';
+
+const {
+	Component,
+	computed,
+	inject
+} = Ember;
+
+export default Component.extend({
+	classNames: ['local-navigation-for-seo'],
+	wikiVariables: inject.service(),
+	localLinks: computed.reads('wikiVariables.localNav'),
+	currentLocalLinks: computed.or('currentLocalNav.children', 'localLinks'),
+	flatNavigationLinks: computed('currentLocalLinks', function () {
+		let deepMap = function (linksList) {
```

Comment:
please extract this method outside component, so it does not have to be created anytime currentLocalLinks is updated.